tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -1,7 +1,7 @@
 # Runtime Resources Visibility
 
 The Runtime Resources Visibility screen lets developers view the details and status of their Kubernetes
```

Comment:
The Runtime Resources Visibility tab shows developers the details and status of their Kubernetes